network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractIdentifiableImpl.java

```diff
@@ -39,6 +39,8 @@ public abstract class AbstractIdentifiableImpl<I extends Identifiable<I>, D exte
 
     private Resource<D> resource;
 
+    private String idIfRemoved;
```

Comment:
```suggestion
    private String removedResourceId;
```
or resourceIdBeforeRemoval ?

Reply:
`idBeforeRemoval` ?

Reply:
Ok !